### PR TITLE
feat : added page for score with correct answer

### PR DESCRIPTION
**Pull request by anupkgurung**
Added page for score.
user can view their final score and scorrect answer

https://kwiizy.vercel.app/

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
code look clean and ui also look good ...

**Review by jeetbhalani15 — APPROVED** [review]
Works fine

File: src/page/Score/Score.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useQuizContext } from "../../context";
+
+const getQuestionByCategory = async (categoryId, setQuestionList) => {
+    try {
+        const { data: { questionList } } = await axios.get(`api/quiz/${categoryId}`);
+        setQuestionList(questionList[0][categoryId]);
+    } catch (error) {
+        console.log(error)
```
Comment (TheAmlanSaikia) [inline src/page/Score/Score.jsx:10]: You can display the error instead of console logging.

**Review by TheAmlanSaikia — APPROVED** [review]
Added a small suggestion. Clean code and good to merge.

**Pull request merged**